src/accountsdb/db.zig

```diff
@@ -2471,49 +2537,67 @@ pub const AccountsDB = struct {
         try writeSnapshotTarWithFields(
             zstd_write_ctx.writer(),
             sig.version.CURRENT_CLIENT_VERSION,
-            StatusCache.default(),
+            StatusCache.EMPTY,
             &snapshot_fields,
             file_map,
         );
         try zstd_write_ctx.finish();
 
+        if (self.gossip) |gossip| { // advertise new snapshot via gossip
+            const push_msg_queue, var push_msg_queue_lg = gossip.push_msg_queue_mux.writeWithLock();
+            defer push_msg_queue_lg.unlock();
+            try push_msg_queue.ensureUnusedCapacity(1);
+
+            const now_ts = sig.time.getWallclockMs();
+            const signed_gossip_data = try sig.gossip.SignedGossipData.initSigned(gossip.my_keypair, .{
+                .SnapshotHashes = .{
+                    .from = Pubkey.fromPublicKey(&gossip.my_keypair.public_key),
+                    .full = .{ .slot = params.target_slot, .hash = full_hash },
+                    .incremental = sig.gossip.data.SnapshotHashes.IncrementalSnapshotsList.EMPTY,
+                    .wallclock = now_ts,
+                },
+            });
+            push_msg_queue.appendAssumeCapacity(signed_gossip_data);
```

Comment:
```suggestion
            const now_ts = sig.time.getWallclockMs();
            const signed_gossip_data = try sig.gossip.SignedGossipData.initSigned(gossip.my_keypair, .{
                .SnapshotHashes = .{
                    .from = Pubkey.fromPublicKey(&gossip.my_keypair.public_key),
                    .full = .{ .slot = params.target_slot, .hash = full_hash },
                    .incremental = sig.gossip.data.SnapshotHashes.IncrementalSnapshotsList.EMPTY,
                    .wallclock = now_ts,
                },
            });

            {
                const push_msg_queue, var push_msg_queue_lg = gossip.push_msg_queue_mux.writeWithLock();
                defer push_msg_queue_lg.unlock();
                try push_msg_queue.append(signed_gossip_data);
            }
```

Reply:
If the `unlock` call blocks for an extended period of time for whatever reason, the `now_ts` could become outdated before we even make it to `append` if we specifically do this.
With that in mind, and the fact that I don't think `initSigned` is particularly slow (meaning I don't think this would realistically reduce contention), I'm not sure this would be an improvement.

Reply:
Will track this along with the discussion about moving the signing to the push queue process; setting the wallclock time there as well may solve this contention.

Reply:
b27f6ac30af6af069e80183fd593e8a50c4386e6